### PR TITLE
SWC-5478: do not set white text, and show maintenance alert for full …

### PR DESCRIPTION
**Pull request by jay-hodgson**
…interval (minus a second for transition)

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java [hunk 1]
```diff
@@ -45,6 +46,7 @@ public void start() {
 		gwt.scheduleFixedDelay(() -> {
 			_getCurrentStatus(this);
 			_getUnresolvedIncidents(this);
+			_getScheduledMaintenance(this);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java:49]: check at reoccurring regular intervals (every minute)

File: src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java [hunk 2]
```diff
@@ -66,16 +68,11 @@ public void onFailure(Throwable caught) {
 			}
 		});
 	}
-	private void checkForScheduledMaintenanceLater() {
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java:69]: No need for special method, now it's called every minute

File: src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java [hunk 3]
```diff
@@ -66,16 +68,11 @@ public void onFailure(Throwable caught) {
...
-		gwt.scheduleExecution(() -> {
-			_getScheduledMaintenance(this);
-		}, INTERVAL_MS);
-	}
 
 	public void showScheduledMaintenance(String name, String scheduledFor, String scheduledUntil) {
 		Date startDate = getDate(scheduledFor);
 		String startTime = dateTimeUtils.getDateTimeString(startDate);
-		popupUtils.showInfo("<a href=\"http://status.synapse.org/\" target=\"_blank\" class=\"color-white\">Maintenance on " + startTime + " - " + name + "</a>", POPUP_DELAY_MS);
+		popupUtils.showInfo("<a href=\"http://status.synapse.org/\" target=\"_blank\"><strong>Maintenance on " + startTime + " - " + name + "</strong></a>", INTERVAL_MS - 1200);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java:75]: No longer white text.  Put in a strong to make it bold

File: src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java [hunk 4]
```diff
@@ -99,61 +96,55 @@ private static native void _getCurrentStatus(SynapseStatusDetector x) /*-{
 					page : @org.sagebionetworks.web.client.SynapseStatusDetector::STATUS_PAGE_IO_PAGE
 				});
 			sp.status({
-					success : function(data) {
-						if (data.status.indicator !== 'none') {
-							// Houston, we have a problem...
-							var description = data.status.description;
-							x.@org.sagebionetworks.web.client.SynapseStatusDetector::showOutage(Ljava/lang/String;)(description);
-						}
+				success : function(data) {
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java:99]: whitespace only

File: src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java [hunk 5]
```diff
@@ -99,61 +96,55 @@ private static native void _getCurrentStatus(SynapseStatusDetector x) /*-{
...
-						filter : 'upcoming',
-						success : function(data) {
-							if (data.scheduled_maintenances[0]) {
-								var scheduledMaintenance = data.scheduled_maintenances[0];
-								var scheduledFor = scheduledMaintenance.scheduled_for;
-								var scheduledUntil = scheduledMaintenance.scheduled_until;
-								var name = scheduledMaintenance.name;
-								// there's a scheduled maintenance. invoke alert with info.
-								x.@org.sagebionetworks.web.client.SynapseStatusDetector::showScheduledMaintenance(Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)(name, scheduledFor, scheduledUntil);
-							} else {
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java:148]: removed else block (checkForScheduledMaintenanceLater())

File: src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java [hunk 6]
```diff
@@ -99,61 +96,55 @@ private static native void _getCurrentStatus(SynapseStatusDetector x) /*-{
...
-								// no scheduled maintenance found, check again later
-								x.@org.sagebionetworks.web.client.SynapseStatusDetector::checkForScheduledMaintenanceLater()();
-							}
-						}
-					});
-		} else {
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java:154]: removed else block (checkForScheduledMaintenanceLater())

File: src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java (hunk 3 above)
Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/SynapseStatusDetector.java:75]: Increase timeout from 5 seconds to about a minute

**Review by xschildw — APPROVED** [review]

**Pull request merged**